common/src/View/wxUtils.cpp

```diff
@@ -53,6 +53,15 @@
 
 namespace TrenchBroom {
     namespace View {
+        DisableWindowUpdates::DisableWindowUpdates(QWidget* widget) :
```

Comment:
I've been deleting usages of the wx update locker rather than porting them to Qt, since it's not needed to avoid flicker in Qt. I guess it doesn't hurt, but I'd just as soon delete them if they're not needed

Reply:
I am using this for performance improvement (out of habit tbh), but we can drop it until we see an actual problem.